pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -950,14 +950,14 @@ func (a *Operator) getOperatorGroupTargets(op *v1.OperatorGroup) (map[string]str
 	} else if selector == nil || selector.Empty() || selector == labels.Nothing() {
 		namespaceSet[corev1.NamespaceAll] = struct{}{}
 	} else {
-		matchedNamespaces, err := a.lister.CoreV1().NamespaceLister().List(selector)
+		matchedNamespaces, err := a.opClient.KubernetesInterface().CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{LabelSelector: selector.String()})
```

Comment:
Can you walk me through the functional difference between these two listers clients? I'm under the impression that both of them are backed by a cache, so we're not hitting the API server directly. If there's no functional difference, maybe we have a bug in the lister implementation?

Reply:
Oh, I thought the opClient doesn't hit the cache.  How can I hit the API directly?

Reply:
@timflannagan I looked at the creation of the `a.opClient.KubernetesInterface()`, again.  I think it's hitting the API server.  Am I misunderstanding?

Reply:
Okay I likely glossed over the code locally too quickly. I think this change is fine then, but it's not entirely clear to me whether this increases the API server load.

/lgtm